R/object_name_linters.R

```diff
@@ -1,3 +1,78 @@
+#' @describeIn linters  Check that object names conform to a single naming style.
+#' @param styles One of \Sexpr[stage=render, results=rd]{lintr:::regexes_rd}.
```

Comment:
param for `styles` states 'one of ...' but the `styles` argument allows multiple style names.

Reply:
Definition for `styles` now reads
```
A subset of \Sexpr[stage=render, results=rd]{lintr:::regexes_rd}. A name should
match at least one of these styles.
```